build.gradle.kts

```
@@ -116,7 +118,7 @@ dependencies {
     compileOnly("me.clip:placeholderapi:2.11.5") // https://www.spigotmc.org/resources/placeholderapi.6245/history
     compileOnly("io.th0rgal:oraxen:1.163.0") // https://www.spigotmc.org/resources/%E2%98%84%EF%B8%8F-oraxen-custom-items-blocks-emotes-furniture-resourcepack-and-gui-1-18-1-21-3.72448/
     compileOnly("com.github.LoneDev6:API-ItemsAdder:3.6.1") // https://www.spigotmc.org/resources/%E2%9C%A8itemsadder%E2%AD%90emotes-mobs-items-armors-hud-gui-emojis-blocks-wings-hats-liquids.73355/updates
-
+    compileOnly("net.countercraft:movecraft:+") //https://hangar.papermc.io/Airship-Pirates/Movecraft
```

```suggestion
    compileOnly("net.countercraft:movecraft:<my-exact-version>") //https://hangar.papermc.io/Airship-Pirates/Movecraft
```
Generally a good idea to use an exact version for a dependency.